### DISCUSSION TITLE
EddieCon 0.1 mini hackathon submissions

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#2208] [Conference Notes]
- Build on the workshop to customise your project
- Submit your repo to win prizes

**Your repo must:**
- Be Open Source
- Use DataStax
- Submitted to this GitHub Discussion by 17:15 BST

This discussion will be unlocking/locking times
- Unlock after the workshop on the 20th August
- Lock at 17:15 BST 20th August

EddieCon website https://www.eddiehub.org/eddiecon-01

**Comment by Amishakumari544** [comment]
We have to submit our repo here??

**Reply by kaiwalyakoparkar** [reply]
yes (github link)

**Comment by kaiwalyakoparkar** [comment]
I tried to create a project which will help Eddie manage his tasks for his birthday :slightly_smiling_face: . I am super happy that I was able to do perform CRUD operations on DataStax Astra with some help of templated frontend.
https://github.com/kaiwalyakoparkar/eddies-birthday-preparation

**Reply by Amishakumari544** [reply]
But I think your code is just saw somewhere 😅 but its awesome 😎

**Reply by kaiwalyakoparkar** [reply]
Yup the front-end is templated. I did not write the frontend part. I wanted to learn astra-db from long time and got a chance to work with it. That made my day. :)

**Comment by Amishakumari544** [comment]
I learned about react native so i added the react native code also which i learned how to do get request in in react native using expo and that was too much of fun.
And for API workshop I just learned and created API by learning from eddie .
https://github.com/Amishakumari544/Workshop-API

**Comment by Prathamesh-Shanbhag** [comment]
This is my first time working with APIs in such detail & I have never done API Testing so using Cucumber & Pactum was a treat!
Here is my project!
👇👇👇
https://github.com/Prathamesh-Shanbhag/Socialy

**Comment by JagjiwanChimkar** [comment]
Can I submit the project now @eddiejaoude

**Reply by Vyvy-vi** [reply]
yes, go on ahead :)

**Comment by Zain-Muiz** [comment]
Thank you @Vyvy-vi , Ill submit mine as well ! Here's Mine,  An API to fetch events with speaker details at Eddiecon 1.0, could definitely improve, But as of now, Here it is ! https://github.com/Zain-Muiz/ApiEddiecon

**Comment by ahamedbasha-n** [comment]
I have created a Web App which helps in fetching the information of the Weather in Different Places. 
Repository - https://github.com/ahamedbasha-n/Weather-App
Languages Used - HTML,CSS, JavaScript.
Thank you so much for the opportunity. Looking forward to work on it more.
